src/completions.ts

```diff
@@ -160,16 +160,18 @@ export abstract class CompletionSourceFuse extends CompletionSource {
         includeScore: true,
         findAllMatches: true,
         ignoreLocation: true,
-        threshold: config.get("completionfuzziness"),
-        minMatchCharLength: 1,
+        threshold: config.get("fuzzy_threshold"),
+        ignoreFieldNorm: true,
+        useExtendedSearch: false,
+        // minMatchCharLength: 3,
     }
 
     // PERF: Could be expensive not to cache Fuse()
     // yeah, it was.
     fuse = undefined
 
     protected lastExstr: string
-    protected sortScoredOptions = false
+    protected sortScoredOptions = true
```

Comment:
I can't think of a good reason why this was ever `false`, so there could well be one

Reply:
Maybe some completions don't want to sort by best match for some reason?

Reply:
but why? :thinking:

Reply:
No idea